.mailmap

```diff
@@ -83,6 +86,8 @@ Gareth Daniel Smith <garethdanielsmith@gmail.com> Gareth Smith <garethdanielsmit
 Georges Dubus <georges.dubus@gmail.com> <georges.dubus@compiletoi.net>
 Graham Fawcett <graham.fawcett@gmail.com> Graham Fawcett <fawcett@uwindsor.ca>
 Graydon Hoare <graydon@pobox.com> Graydon Hoare <graydon@mozilla.com>
+Guillaume Gomez <guillaume1.gomez@gmail.com>
```

Comment:
No (try removing it and we get duplicates). _Maybe_ there's a more clever way to condense all of Guillaume's contributions with just two total .mailmap directives rather than three?—but finding it doesn't seem like a priority.